frontend/src/index.ts

```diff
@@ -642,6 +642,7 @@ export class App extends LiteElement {
           .viewStateData=${this.viewState.data}
           .params=${this.viewState.params}
           orgId=${orgId}
+          orgStorageQuotaReached="false"
           orgPath=${orgPath.split(orgId)[1]}
```

Comment:
See comment in `frontend/src/pages/org/index.ts` for rationale:
```suggestion
          orgPath=${orgPath.split(orgId)[1]}
```